README.md

```diff
@@ -20,54 +21,57 @@ Please consider anything which is not published on https://eips.ethereum.org/ as
 
 And please consider anything published at https://eips.ethereum.org/ with a status of "draft" as an incomplete draft.
 
-# Validation
+## Automerger
 
-EIPs must pass some validation tests.  The EIP repository ensures this by running tests using [html-proofer](https://rubygems.org/gems/html-proofer) and [eipv](https://github.com/lightclient/eipv).
+This repository contains an "auto merge" feature to ease the workload for EIP editors. Pull requests to any EIP will be auto-merged if that EIP's authors will approve the PR on GitHub. This is handled by the [EIP-Bot](https://github.com/ethereum/EIP-Bot).
+
+## Validation
+
+Pull requests in this repository must pass automated validation checks:
+
+* HTML formatting and broken links are [checked](https://github.com/ethereum/EIPs/blob/master/.travis-ci.sh) using [html-proofer](https://rubygems.org/gems/html-proofer).
+* EIP front matter and formating are [checked](https://github.com/ethereum/EIPs/blob/master/.github/workflows/auto-merge-bot.yml) using [EIP Validator](https://github.com/lightclient/eipv).
 
 It is possible to run the EIP validator locally:
 ```sh
 cargo install eipv
 eipv <INPUT FILE / DIRECTORY>
 ```
 
-# Automerger
-
-The EIP repository contains an "auto merge" feature to ease the workload for EIP editors.  If a change is made via a PR to a draft EIP, then the authors of the EIP can GitHub approve the change to have it auto-merged. This is handled by the [EIP-Bot](https://github.com/ethereum/EIP-Bot).
+## Build the status page locally
 
-# Local development
-
-## Prerequisites
+### Install prerequisites
 
 1. Open Terminal.
 
 2. Check whether you have Ruby 2.1.0 or higher installed:
 
-```sh
-$ ruby --version
-```
+   ```sh
+   ruby --version
+   ```
 
 3. If you don't have Ruby installed, install Ruby 2.1.0 or higher.
 
 4. Install Bundler:
 
-```sh
-$ gem install bundler
-```
+   ```sh
```

Comment:
Nit: why do these need to be indented?

Reply:
This renders so that the code fence is part of the step

Reply:
Ah!

Before: <img width="853" alt="Screenshot" src="https://user-images.githubusercontent.com/20340/153039433-7e8dedef-ee45-412d-b246-e6844906cd0e.png">

After: <img width="862" alt="Screenshot" src="https://user-images.githubusercontent.com/20340/153039339-3a60c587-a5d4-4136-8abf-276a6c1565b5.png">